config/jobs/kubernetes/sig-auth/serviceaccount-admission-controller-migration-config.yaml

```diff
@@ -0,0 +1,27 @@
+periodics:
+- name: serviceaccount-admission-controller-migration
+  annotations:
+    testgrid-dashboards: sig-auth-gce
+  labels:
+    preset-service-account: "true"
+    preset-k8s-ssh: "true"
+  decorate: true
+  interval: 2h
+  spec:
+    containers:
+      - image: gcr.io/k8s-testimages/kubekins-e2e:v20200918-58bdf8d-master
+        args:
+        - --timeout=920
+        - --bare
+        - --scenario=kubernetes_e2e
+        - --
+        - --check-leaked-resources
+        - --check-version-skew=false
+        - --env=STORAGE_MEDIA_TYPE=application/vnd.kubernetes.protobuf
+        - --extract=ci/latest
+        - --extract=ci/latest-1.19
```

Comment:
what will this do once ci/latest is 1.21? Should we be using stable1 markers? cc @justaugustus

Reply:
use ci/latest-1.19 and ci/latest so that the hash and coredns fix are included. we can change them later if appropriate.

Reply:
Sounds good, let's make sure that's in an issue to track

Reply:
done. https://github.com/kubernetes/test-infra/issues/19575